dotenv.example

```diff
@@ -16,6 +16,8 @@ TWO_FACTOR_FORCE_OTP_ADMIN=no
 
 # CSP_REPORT_ONLY=yes
 
+# HYPOTHESIS_PROFILE=ci
+
```

Comment:
Maybe add a line to explain devs that the `ci` profile should not be your default preference when developing with hypothesis tests.

Reply:
done!